packages/fx-core/templates/core/v3Migration/csharp.app.local.yml

```diff
@@ -51,14 +51,20 @@ provision:
 
 configureApp:
 {{#if activePlugins.fx-resource-frontend-hosting}}
-  - uses: file/updateEnv # Generate env to .env file
+  - uses: script # Set env for local launch
+    name: Set {{placeholderMappings.tabDomain}} for local launch
     with:
-      envs:
-        {{placeholderMappings.tabDomain}}: localhost:44302
-        {{placeholderMappings.tabEndpoint}}: https://localhost:44302
-        {{placeholderMappings.tabIndexPath}}: '#'
+      run: ::set-output {{placeholderMappings.tabDomain}}=localhost:44302
```

Comment:
echo "::set-output xxx"
See this change: https://github.com/OfficeDev/TeamsFx/pull/8048

Reply:
Good catch... I need to adjust all templates as well

Reply:
I raised a PR to update the templates (https://github.com/OfficeDev/TeamsFx/pull/8067) because it blocks our E2E cases.